src/Forms/Checkbox.js

```diff
@@ -28,6 +28,8 @@ const Checkbox = React.forwardRef(({ checked, className, defaultChecked, disable
         'fd-checkbox'
     );
 
+    const isChecked = typeof checked === 'undefined' ? defaultChecked : checked;
```

Comment:
Couldn't the same thing happen if both `checked` and `defaultChecked` are undefined?  Wouldn't you just end up with undefined being sent to the `<input>`?